ImperatorToCK3/Source/Configuration/Configuration.cpp

```diff
@@ -49,8 +49,23 @@ void Configuration::registerKeys()
 		});
 	registerKeyword("ImperatorDeJure", [this](const std::string& unused, std::istream& theStream) {
 		const commonItems::singleString deJureString(theStream);
-		imperatorDeJure = static_cast<IMPERATOR_DE_JURE>(stoi(deJureString.getString()));
-		Log(LogLevel::Info) << "CK3 de iure set to: " << deJureString.getString();
+		try
+		{
+			imperatorDeJure = static_cast<IMPERATOR_DE_JURE>(stoi(deJureString.getString()));
+			Log(LogLevel::Info) << "CK3 de iure set to: " << deJureString.getString();
+		}
+		catch (const std::invalid_argument& ia)
```

Comment:
Just catch std::exception and log what the deJureString was. Further detail won't help debug.